packages/shared/src/components/tooltips/InteractivePopup.tsx

```diff
@@ -37,6 +38,7 @@ const positionClass: Record<InteractivePopupPosition, string> = {
   leftStart: classNames(leftClass, startClass),
   leftCenter: classNames(leftClass, centerClassY),
   leftEnd: classNames(leftClass, endClass),
+  screen: 'inset-0 w-screen h-screen',
```

Comment:
At line 32, there are certain classes for `center` position that we don't need. So we introduce a specific one for the case we have now.

Reply:
Ah right, something modal ofcourse does, good to know for future I guess :)